apps/meteor/client/providers/CallProvider/CallProvider.tsx

```diff
@@ -161,20 +163,36 @@ export const CallProvider: FC = ({ children }) => {
 		roomCoordinator.openRouteLink('v', { rid });
 	}, []);
 
+	const findOrCreateVisitor = useCallback(
+		async (caller: ICallerInfo): Promise<ILivechatVisitor> => {
```

Comment:
```suggestion
		async (caller: ICallerInfo): Promise<Serialized<ILivechatVisitor>> => {
```

This should solve all the problems and we can remove all type assertions.